lib/datasource/npm/npmrc.ts

```diff
@@ -112,8 +112,7 @@ export function resolvePackage(packageName: string): PackageResolution {
     !authInfo &&
     npmrc &&
     npmrc._authToken &&
-    registryUrl.replace(regEx(/\/?$/), '/') ===
-      npmrc.registry?.replace(/\/?$/, '/') // TODO #12070
+    registryUrl.replace(/\/?$/, '/') === npmrc.registry?.replace(/\/?$/, '/') // TODO #12070
```

Comment:
why this revert?

Reply:
Both regex are same so it is illogical how one could work with re2 and not the other.